glass/shells.py

```diff
@@ -625,14 +716,18 @@ def combine(
         Find weights for a given function.
 
     """
-    return sum(
+    return (
```

Comment:
`sum(<x>)` is equivalent to `np.sum(<x>, axis=0)` and is more accurately showing what is happening + fixes `mypy`